spec/ruby/language/symbol_spec.rb

```diff
@@ -72,8 +72,16 @@
     }
   end
 
-  it "can contain null in the string" do
-    eval(':"\0" ').inspect.should == ':"\\x00"'
+  ruby_version_is "".."2.6" do
+    it "can contain null in the string" do
+      eval(':"\0" ').inspect.should == ':"\\x00"'
```

Comment:
This could be rewritten to be version-independent, like `.to_s.bytes[0].should == 0`

Reply:
Very good point.